blueprints/third-party-solutions/f5-bigip/f5-bigip-ha-active/variables.tf

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "f5_vms_dedicated_config" {
+  description = "The F5 VMs configuration. The map keys are the zones where the VMs are deployed."
+  type = map(object({
```

Comment:
nit: I understand the naming you used but don't love it :)

This variable holds the network configuration for VMs, can we rename as `instance_vpc_configs` or somesuch? and the one below as `instance_vm_config`?

TL;DR I would like variable names to make it immediately obvious that one configures network-related attributes for *each* VM, the other the underlying instance configuration for all.

If you really like this approach though, we could slightly change `f5_vms_shared_config` to `instance_shared_config`, and use `instance_vpc_configs` for the other.

WDYT?

Reply:
resolved in 1:1